src/SFA.DAS.AdminService.Web/Infrastructure/RoatpClients/RoatpApplicationApiClient.cs

```diff
@@ -153,7 +153,15 @@ public async Task<GatewayPageAnswer> GetGatewayPageAnswer(Guid applicationId, st
 
         public async Task<GatewayCommonDetails> GetPageCommonDetails(Guid applicationId, string pageId, string userName)
         {
-            return await Get<GatewayCommonDetails>($"Gateway/Page/CommonDetails/{applicationId}/{pageId}/{userName}");
+            try
+            {
+                return await Get<GatewayCommonDetails>($"Gateway/Page/CommonDetails/{applicationId}/{pageId}/{userName}");
+            }
+            catch (Exception ex)
```

Comment:
Should all exceptions result in the same shutter page?  There are parts of this call which aren't related to external apis,

Reply:
Might it be better for the apply api to return a particular response code (service unavailable maybe?) when an external API is down and show the error page based on that instead?

Reply:
I went for exceptions rather than response codes, because we'd need to handle exceptions anyway for timeouts, i.e. the external service 4 hops away doesn't respond in time

Reply:
Fair enough.  Can you merge in @martin-hornsby 's branch and catch the new exception thrown?  That way and non-api exceptions (not that there are ever likely to be any!) will be handled in the default way.

Reply:
Done